File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/fragments/drag_and_drop/util/EmptyTargetItem.java
package de.dimedis.mobileentry.fragments.drag_and_drop.util;

import de.dimedis.mobileentry.fragments.drag_and_drop.model.Item;

public class EmptyTargetItem extends Item {
    public EmptyTargetItem() {
        functionName = "";
        ItemString = "";
    }

    public EmptyTargetItem(int position) {
        functionName = "";
        ItemString = "";

        setPosition(position);
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/fragments/drag_and_drop/util/EmptyTargetItem.kt
package de.dimedis.mobileentry.fragments.drag_and_drop.util

import de.dimedis.mobileentry.model.Item

class EmptyTargetItem : Item {
    constructor() {
        functionName = ""
        ItemString = ""
    }

    constructor(position: Int) {
        functionName = ""
        ItemString = ""
        setPosition(position)
    }
}